spec/features/admin/enterprises/terms_and_conditions_spec.rb

```diff
@@ -22,30 +23,32 @@ def go_to_business_details
         end
       end
 
-      let(:white_pdf_file_name) { Rails.root.join("app", "assets", "images", "logo-white.pdf") }
-      let(:black_pdf_file_name) { Rails.root.join("app", "assets", "images", "logo-black.pdf") }
+      let(:white_pdf_file_name) { Rails.root.join("app/assets/images/logo-white.pdf") }
+      let(:black_pdf_file_name) { Rails.root.join("app/assets/images/logo-black.pdf") }
 
       before do
         # Create fake PDFs from PNG images
-        FileUtils.cp(Rails.root.join("app", "assets", "images", "logo-white.png"), white_pdf_file_name)
-        FileUtils.cp(Rails.root.join("app", "assets", "images", "logo-black.png"), black_pdf_file_name)
-  
+        FileUtils.cp(Rails.root.join("app/assets/images/logo-white.png"), white_pdf_file_name)
```

Comment:
Rubocop complaint about this? I thought the first version would be better...

Reply:
yes, rubocop asked for this one :+1: